src/mumble/UserModel.cpp

```diff
@@ -990,6 +990,7 @@ void UserModel::removeUser(ClientUser *p) {
 	updateOverlay();
 
 	delete p;
+
```

Comment:
Unrelated change, do you want to keep it?

Reply:
No. That's just a remnant of my debugging statements xD